pom.xml

```diff
@@ -110,6 +110,7 @@
     <spark.bundle.hive.shade.prefix></spark.bundle.hive.shade.prefix>
     <utilities.bundle.hive.scope>provided</utilities.bundle.hive.scope>
     <utilities.bundle.hive.shade.prefix></utilities.bundle.hive.shade.prefix>
+    <argLine>-Xmx1024m -XX:MaxPermSize=256m</argLine>
```

Comment:
We need more memory only for integration tests, any reason to increase memory here ?

Reply:
This is an unrelated change but is required to run jacoco. Currently jacococ plugin does not run on master branch.

I will separate this into another PR.